engine/execution/rpc/engine.go

```diff
@@ -14,10 +14,14 @@ import (
 	"github.com/rs/zerolog"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
+	_ "google.golang.org/grpc/encoding/gzip"
 	"google.golang.org/grpc/status"
 
 	"github.com/onflow/flow-go/consensus/hotstuff"
 	"github.com/onflow/flow-go/engine"
+	_ "github.com/onflow/flow-go/engine/common/grpc/compressor/deflate"
+	_ "github.com/onflow/flow-go/engine/common/grpc/compressor/lz4"
+	_ "github.com/onflow/flow-go/engine/common/grpc/compressor/snappy"
```

Comment:
can you add a comment about why these are required. otherwise, I worry someone may remove them and break compression